lib/model/migrate.js

```diff
@@ -25,5 +25,12 @@ const withDatabase = (config) => (mutator) => {
 // Given a database, initiates migrations on it.
 const migrate = (db) => db.migrate.latest({ directory: `${__dirname}/migrations` });
 
-module.exports = { connect, withDatabase, migrate };
+// Checks for pending migrations and returns an exit code of 1 if any are
+// still pending/unapplied (e.g. automatically running migrations just failed).
+const checkMigrations = (db) => db.migrate.list({ directory: `${__dirname}/migrations` })
+  .then((res) => (res[1].length === 0
+    ? process.exit(0)
+    : process.exit(1)));
```

Comment:
This area is new to me, but I'm reading [here](https://nodejs.dev/learn/how-to-exit-from-a-nodejs-program) that `process.exit()` "ungracefully" terminates the program. `withDatabase()` will want to call `db.destroy()` after running `checkMigrations`, so if there's another way to terminate, that seems preferable. (That link seems to suggest setting `process.exitCode`.)